### ISSUE TITLE
i train  databse,but there is a qeustion,how can l solve it ,please,help me 

### ISSUE THREAD

**Issue by dazhangzhang** [jwyang/faster-rcnn.pytorch#478]
[session 1][epoch  1][iter 3800/9622] loss: nan, lr: 1.00e-05
                        fg/bg=(256/0), time cost: 20.477921
                        rpn_cls: nan, rpn_box: nan, rcnn_cls: nan, rcnn_box nan
[session 1][epoch  1][iter 3900/9622] loss: nan, lr: 1.00e-05
                        fg/bg=(256/0), time cost: 20.650007
                        rpn_cls: nan, rpn_box: nan, rcnn_cls: nan, rcnn_box nan
[session 1][epoch  1][iter 4000/9622] loss: nan, lr: 1.00e-05
                        fg/bg=(256/0), time cost: 20.547670
                        rpn_cls: nan, rpn_box: nan, rcnn_cls: nan, rcnn_box nan

**Comment by a527533557** [comment]
same problem!  did you solve it?

**Comment by jinggew** [comment]
have you solved it?

**Comment by zychen2016** [comment]
@jinggew 
@dazhangzhang 
@a527533557 

have you solved it?

**Comment by Complicateddd** [comment]
I think that some bugs happen in roibatchLoader.py if you use your own dataset with flip or corp operation
you can skip the nan iter with followed codes in train.py
with torch.no_grad():
                  im_data.resize_(data[0].size()).copy_(data[0])
                  im_info.resize_(data[1].size()).copy_(data[1])
                  gt_boxes.resize_(data[2].size()).copy_(data[2])
                  num_boxes.resize_(data[3].size()).copy_(data[3])
                  keep1=gt_boxes[:,:,0]>gt_boxes[:,:,2]
                  keep2=gt_boxes[:,:,1]>gt_boxes[:,:,3]
            if torch.nonzero(keep1==1).view(-1).numel()>0 or torch.nonzero(keep2==1).view(-1).numel()>0:
                continue
may be this can help u
